.vscode/settings.json

```diff
@@ -17,4 +17,5 @@
   },
   "editor.tabSize": 2,
   "files.insertFinalNewline": true,
+  "serverlessIDE.telemetry.enableTelemetry": false,
```

Comment:
I can remove this if you want, but I found https://marketplace.visualstudio.com/items?itemName=ThreadHeap.serverless-ide-vscode

Note that this will only work **if your file is serverless.yml**, so `serverless.example.yml` doesn't pick up the features. I had to make a temp file `serverless.yml` in the project

Reply:
I'd prefer to remove this, and if you could actually delete this file altogether while you're at it and add `.vscode/` to .gitignore, thx